epubmaker_postprocessing.rb

```diff
@@ -257,9 +257,7 @@ def sendAlertMail(epubcheck_output, testing_value_file, message, logkey='')
 
 # hide toc entry in html toc
 # fix title page text in html toc
-htmlcontents = readFile("#{oebps_dir}/toc01.html", 'read_htmlcontents')
-replace = hideTOC_fixTPtext(htmlcontents, 'hide_toc-entry_in_toc__fix_tptext_in_toc')
-overwriteFile("#{oebps_dir}/toc01.html", replace, 'write_new_htmlcontents')
+localRunNode(epubmakerpostprocessingTOCjs, "#{oebps_dir}/toc01.html", 'epubmaker_postprocessing_TOC_js')
```

Comment:
Create a new variable for the html file with File.join, instead of coding the path directly into the method.